### PR TITLE
Wuzzuf feature Added

### PR DESCRIPTION
**Pull request by msabry1**
## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [ yes] Have you adhered to the repository's defined coding convention rules?
- [ yes] Have you updated the 'documentation.md' file with the method/function documentation?
- [ yes in the bottom] Have you sent a message along with the result or response?
- [ yes] Have you used the try-catch technique?
- [yes ] Has the method/class been added to the documentation (md file)?

## Screenshots

image

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ per user.followers()
 - [Flyrobu](https://github.com/Clueless-Community/scrape-up/blob/main/documentation.md#flyrobu)
 - [HealthGrades](https://github.com/Clueless-Community/scrape-up/blob/main/documentation.md#healthgrades)
 - [IMDB](https://github.com/Clueless-Community/scrape-up/blob/main/documentation.md#imdb)
-
+- [Wuzzuf](https://github.com/Clueless-Community/scrape-up/blob/main/documentation.md#Wuzzuf)
```
Comment (nikhil25803) [inline documentation.md:47]: In the `documentation.md` file, we maintain the documentation of the modules that are in the deployed version of the package only. 

Remove it from here and include it in `dev-documentation.md` file. Also do not just add link to the location. Also document you modules like this
```js
## Dictionary.com

First create an object of class `Dictionary`.


| Methods          | Details                                                                                      |
| ---------------- | -------------------------------------------------------------------------------------------- |
| `.get_word_of_the_day()`       | Returns the word of the day. |
| `.word_of_the_day_definition()`    | Returns the definition of the word of the day.
--------

```

File: documentation.md [hunk 2]
```diff
@@ -733,3 +733,34 @@ boxoffice = imdb.BoxOffice()
...
+
+| Methods               | Details                                                                                             |
+| --------------------- | --------------------------------------------------------------------------------------------------- |
+| `.filter_job()`        | Apply filters such as job title, country, city, and range of years of experience.                   |
+| `.fetch_jobs()`        | Fetch job listings from the website based on the applied filters, across multiple pages.            |
+
+```python
+from scrap-up import wuzzuf
+```
+### How to use :
```
Comment (nikhil25803) [inline documentation.md:749]: Remove this how to use section. You can see how other modules are documented. 
You can implement the same way.

File: src/scrape_up/wuzzuf/stringdecorator.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+def string_decorator(func):
```
Comment (nikhil25803) [inline src/scrape_up/wuzzuf/stringdecorator.py:1]: No need of this, just return None in any case the scraper is not able to return the result.
We do create files only for modules.

File: src/scrape_up/wuzzuf/wuzzuf.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
+import requests
+from bs4 import BeautifulSoup
+from time import sleep
+from stringdecorator import string_decorator
+
+
+class Jobs:
+    """
```
Comment (nikhil25803) [inline src/scrape_up/wuzzuf/wuzzuf.py:8]: I renamed the class, but I want you to rephrase the docstring, you can check modules like `GitHub` to see how docstrings are documented.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Hey @msabry1, your code is working fine and returning the desired result. Although few minor changes are required. 

Please address them.

**Comment by msabry1** [comment]
@nikhil25803 ok you can review it now 
I changed all you said

**Review by nikhil25803 — APPROVED** [review]

**Comment by nikhil25803** [comment]
Great work @msabry1  !!
Do not forget to
+ Share your contribution on socials and tag Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Pull request merged**